test/CacheFactoryTest.php

```diff
@@ -41,12 +41,12 @@ public function testFileSystemCacheConstructor() : void
             ],
         ];
 
-        $container = $this->prophesize(ContainerInterface::class);
-        $container->has('config')->willReturn(true);
-        $container->get('config')->willReturn($config);
+        $container = $this->getMockBuilder(ContainerInterface::class)->onlyMethods(['has', 'get'])->getMock();
```

Comment:
`$this->createMock()` would suffice: the `ContainerInterface` only has those methods

Reply:
Addressed with bbe09841c12999e74a99b7af5d99804871fc90f2